src/sentry/sentry_metrics/indexer/postgres.py

```diff
@@ -34,7 +36,18 @@ def bulk_record(self, strings: List[str]) -> Mapping[str, int]:
         mapped_result: MutableMapping[str, int] = {r.string: r.id for r in cache_results}
 
         unmapped = set(strings).difference(mapped_result.keys())
-        new_mapped = self._bulk_record(unmapped)
+        if not unmapped:
+            metrics.incr("sentry_metrics.indexer.memcache.hit")
+            return mapped_result
+        else:
+            percent_unmapped = int((len(unmapped) / len(strings)) * 100)
+            metrics.incr(
+                "sentry_metrics.indexer.memcache.miss",
+                tags={"percent_missed": percent_unmapped},
+            )
```

Comment:
I maybe am misunderstanding Datadog, but it seems like you might want to separately emit misses and total fetches as a metric so that the counts of misses and fetches between hosts can be combined for an aggregate percent missed